data_generation/IPCC2006_PRIMAP.py

```diff
@@ -122,6 +123,7 @@ def main():
     ipcc2006_primap.canonical_top_level_category = ipcc2006_primap["0"]
 
     ipcc2006_primap.to_yaml(OUTPATH)
+    ipcc2006_primap.to_python(OUTPATH_PY)
```

Comment:
This is somewhat redundant given that the cached python is also made from the yaml files via the makefile, or am I missing something?

Reply:
Yes, good catch! (I overlooked this one while removing it in the others.)